lib/htmlValidator.js

```diff
@@ -42,47 +42,51 @@ module.exports = function(app, callback) {
       params.htmlValidator.validator = 'http://localhost:' + (params.htmlValidator.port ? params.htmlValidator.port : '8888');
       options = params.htmlValidator;
     }
-    // spawn validator child process
-    promise = (() => {
-      return new Promise((resolve, reject) => {
-        validatorProcess = spawn(
-          'java', ['-Xss1024k', '-cp', vnu, 'nu.validator.servlet.Main', params.htmlValidator.port ? params.htmlValidator.port : '8888']
-        );
-
-        console.log('\u23F3  ' + ('Starting HTML validator...').bold.yellow);
-
-        validatorProcess.stdout.on('data', (data) => {
-          if (`${data}`.includes('Initialization complete')) {
-            clearTimeout(validatorTimeout);
-            console.log('✔️  ' + ('Validator initialization completed successfully. HTML validator listening on port: ' + params.htmlValidator.port).bold.green);
-            resolve();
-          }
-        });
 
-        validatorProcess.stderr.on('data', (data) => {
-          if (`${data}`.includes('No Java runtime present, requesting install.')) {
-            console.error('❌  ' + ('There was an error in authentication. Please make sure to have Java installed.').red);
-            reject();
-          }
-        });
+    // only run validator if in development mode
+    if (process.env.NODE_ENV === 'development') {
```

Comment:
the proper check for this i think is higher up the stack

much simpler to strip all checks for dev mode from this file, then in sourceParams to override the setting

so, set enableValidator to false here: https://github.com/rooseveltframework/roosevelt/blob/5c3beab359378041a2795c85648bab5fe64f87eb/lib/sourceParams.js#L173

then remove all checks for dev mode from htmlValidator.js